modules/ossm-security-mtls.adoc

```diff
@@ -35,14 +34,14 @@ You can also configure mTLS for individual services or namespaces by creating a
 
 [source,yaml]
 ----
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
+apiVersion: security.istio.io/v1beta1
```

Comment:
Line 33 says we're creating a policy, but this is a PeerAuthentication resource.